src/invocation/ClientConnection.ts

```diff
@@ -19,6 +19,10 @@ class ClientConnection {
         this.lastRead = 0;
     }
 
+    getLocalAddress() {
+        return new Address(this.socket.localAddress, this.socket.localPort);
```

Comment:
Local address points to the address of the network interface from which the outbound connection is made, right?